README.md

```diff
@@ -155,6 +155,13 @@ sbtix.buildSbtProgram {
 }
 ```
 
+#### `nix` Extra Attribute
+
+Source dependencies do not need locking. Originally this was implemented by completely ignoring dependencies with `nix` [extra attribute](https://www.scala-sbt.org/1.x/docs/Library-Management.html#Extra+Attributes), but this is not necessary anymore.
```

Comment:
It does still exist, which is what I wanted to document.
I've reduced it to more of a statement of fact.
Maybe my approach is too conservative and we should rip it out?

-----

#### `nix` Extra Attribute

By adding the `nix` [extra attribute](https://www.scala-sbt.org/1.x/docs/Library-Management.html#Extra+Attributes), `sbtix` will ignore the dependency for the purpose of locking.

This used to be the only mechanism for handling local dependencies, but is now a legacy solution and/or escape hatch.

Reply:
ah right. I guess a reason to keep it would be so you could disambiguate between a nix-built dependency and the same version from a repo somewhere? I don't feel strongly either way.